Tests/SwiftDocCTests/Test Resources/Inheritance.symbols.json

```diff
@@ -237,7 +237,7 @@
                 "character" : 35
               }
             },
-            "text" : "The index of this instance."
+            "text" : "@Image a known directive"
```

Comment:
Why do we need to modify this test?

Reply:
reverted this change and updated the @returns directive to have a known directive.